napari_cellseg3d/plugin_model_training.py

```diff
@@ -186,16 +186,17 @@ def __init__(
 
         self.zip_choice = ui.make_checkbox("Compress results")
 
-        self.validation_percent_choice = ui.make_combobox(["80%", "90%"])
+        self.validation_percent_choice = ui.DropdownMenu(["80%", "90%"])
```

Comment:
We probably should make this a little more flexible.

Reply:
Done, replaced with a spinbox from 10 to 90%